# BSpline parameters
trajectory_generator:

  time_scale: .2
  degree: 4

  # Use the current position and velocity for the first knot point
  # if specified, the trajectory will be made one knot longer.
  prepend_current_state: False

  # if empty, the node will look for waypoints coming in on
  # the waypoints topic, otherwise, waypoints given here will be used
  static_waypoints:
        [[0.0, 0.0, 0.0],
        [30.0, 0.0, -5.0],
        [50.0, 0.0, -10.0],
        [70.0, -5.0, -10.0],
        [80.0, -20.0, -5.0],
        [80.0, -25.0, 0.0]]
